README.md

```diff
@@ -1,23 +1,19 @@
-This is a Cookiecutter template for creating Overleaf LaTex paper repositories using Billinge group standards.
+This is a Cookiecutter template used in `scikit-package` project to create a configured manuscript folder. Please visit the latest developments and the official documentation [here](https://scikit-package.github.io/scikit-package/).
 
 # Features:
 
 - Auto-generates the repository name, paper title, and one author name.
-- Uses the IUCR LaTeX template (`iucr.bst`, `iucrit.bst`, `iucrjournals.cls`, `fig1.png`).
-- Includes dynamically generated files `cmds-general.tex`, `cmds-programs.tex`, and `packages.tex` from https://github.com/Billingegroup/latex-headers.
-- Includes empty .bib files: `bg-pdf-standards.bib`, `billinge-group-bib.bib`, `hand-coded.bib`, and `repo_name.bib`.
+- Provides the IUCR LaTeX template (`iucr.bst`, `iucrit.bst`, `iucrjournals.cls`, `fig1.png`).
```

Comment:
"By default provides...."  But please check. Isn't the default actually the plain article template?

then add at the end.

"Additional templates are available, for example, the IUCR LaTeX template (`iucr.bst`, `iucrit.bst`, `iucrjournals.cls`, `fig1.png`).   If there is an addional template that you would like to add, please make a PR with this addition to this repository. If you are not familiar with GitHub workflows, please reach out to us with the files and we can do that for you."